### ISSUE TITLE
Reduce the Relational Nature of OVAL

### ISSUE THREAD

**Issue by djhaynes** [scapcommunity/XCCDF#6]
**Challenge:** OVAL Definitions documents are structured so that Definitions, Tests, Object, States, and Variables are all grouped together. This was done to maximize reusability of individual constructs. However, due to the relational nature of OVAL (i.e., references between constructs), this approach makes navigating, comprehending, and editing an OVAL document difficult.

**Proposal:** Consider an approach that makes Definitions self-contained with its corresponding Tests, Objects, States, and Variables since construct reusability seems less of a priority than originally thought.

**Comment by DavidRies** [comment]
I agree with the challenge/difficulty reading large OVAL documents. IMO, reorganizing OVAL into self-contained definitions just moves the problem to a different use case (authoring, evaluating). And, I think that all of these use cases are going to require tooling to do anything with a large OVAL document, no matter how it's organized. It's certainly worth discussion, but my immediate reaction is that the tooling initiatives are more promising than reorganizing the current format.

**Comment by djhaynes** [comment]
This requires further discussion as to whether or not language changes are required to address this problem.
